README.md

```diff
@@ -274,6 +278,16 @@ To create your own custom theme, please refer [theming wiki](https://github.com/
     <tr><td>pokemon-colorscripts-git</td><td>display pokemon sprites</td></tr></table>
 </td></tr></table>
 
+<table><tr><td>
```

Comment:
i think its better to add this optional custom_apps part in wiki section with detailed explanation...
Not sure if new users will understand it right away...

Reply:
I was thinking about that before the last force pushed and yeah, thanks work too.
You could clear these optional pkgs from the custom_apps.lst whenever you want 😃